src/Adaptor/SentryAdaptor.php

```diff
@@ -176,4 +197,18 @@ public static function get_opts(): array
         return $opts;
     }
 
+    public function startTransaction($transactionName)
+    {
+        $hub = SentrySdk::getCurrentHub();
+        $options = $hub->getClient()->getOptions();
+        $rate = $options->getTracesSampleRate();
+
+        $samplingContext = DynamicSamplingContext::fromOptions($options, $this->getContext());
+        $source = TransactionSource::url();
+        $metadata = new TransactionMetadata($rate, $samplingContext, $source);
+
+        $transaction_context = new TransactionContext($transactionName, null, $metadata);
+        $transaction = $hub->startTransaction($transaction_context);
+        return $transaction;
+    }
```

Comment:
@clodeindustrie this adaptor is not a middleware. What's the intended use of this `startTransaction` here?

Reply:
This is now resolved